tools/ossf_best_practices/silver_criteria.md

```diff
@@ -0,0 +1,321 @@
+
+_Possible answers: Met/Unmet_
+
+> The project SHOULD have a "bus factor" of 2 or more. (URL required)
+
+_Possible answers: Met/Unmet_
+
+## Documentation
+
+> The project MUST have a documented roadmap that describes what the project intends to do and not do for at least the next year. (URL required)
+
+_Possible answers: Met/Unmet_
+
+The project MUST include documentation of the architecture (aka high-level design) of the software produced by the project. If the project does not produce software, select "not applicable" (N/A). (URL required)
+
+_Possible answers: Met/Unmet/NA_
+
+> The project MUST document what the user can and cannot expect in terms of security from the software produced by the project (its "security requirements"). (URL required)
+
+_Possible answers: Met/Unmet_
+
+> The project MUST provide a "quick start" guide for new users to help them quickly do something with the software. (URL required)
+
+_Possible answers: Met/Unmet/NA_
+
+
+The project MUST make an effort to keep the documentation consistent with the current version of the project results (including software produced by the project). Any known documentation defects making it inconsistent MUST be fixed. If the documentation is generally current, but erroneously includes some older information that is no longer true, just treat that as a defect, then track and fix as usual.
+
+_Possible answers: Met/Unmet/NA_
+
+
+The project repository front page and/or website MUST identify and hyperlink to any achievements, including this best practices badge, within 48 hours of public recognition that the achievement has been attained. (URL required) 
+
+_Possible answers: Met/Unmet_
+
+
+## Accessibility and internationalization
+
+> The project (both project sites and project results) SHOULD follow accessibility best practices so that persons with disabilities can still participate in the project and use the project results where it is reasonable to do so.
+
+_Possible answers: Met/Unmet/NA_
+
+> The software produced by the project SHOULD be internationalized to enable easy localization for the target audience's culture, region, or language. If internationalization (i18n) does not apply (e.g., the software doesn't generate text intended for end-users and doesn't sort human-readable text), select "not applicable" (N/A)
+
+_Possible answers: Met/Unmet/NA_
```

Comment:
I think is N/A for Nodejs itself, but the documentation and the website has support for i18n

```suggestion
> The software produced by the project SHOULD be internationalized to enable easy localization for the target audience's culture, region, or language. If internationalization (i18n) does not apply (e.g., the software doesn't generate text intended for end-users and doesn't sort human-readable text), select "not applicable" (N/A)

**NA**
```

Reply:
If the website and docs have localization this should be met @ovflowd

Reply:
Resolved in 245dd9656bf16b3378a1af3816ef7aeb58b2ac69